features/mbedtls/crypto/targets/TARGET_PSA/mbed_lib.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "mbed-crypto",
+    "macros": ["MBEDTLS_PSA_HAS_ITS_IO"]
+}
```

Comment:
- What else should we place into the `TARGET_PSA` folder from Mbed Crypto?
- What significance does the "name" field have? Do we need it?
- Nothing is going to use `MBEDTLS_PSA_HAS_ITS_IO` yet, so this PR depends on https://github.com/ARMmbed/mbed-os/pull/8723 (and maybe we should move this commit there via cherry-pick, to lessen the number of dependent PRs a bit)

Reply:
@Patater The name field is required and namespaces any configuration you have.

Reply:
@Patater , all Mbed Crypto sources shall reside inside `TARGET_PSA` folder, see #8723 .
This way we achieve fine grained control over our roll-out plan.
Only targets which have `PSA` label will enjoy of new functionality. We may revisit our PSA configuration plan in later releases.
Build system will skip `TARGET_PSA` folders while scanning for resources, and will not apply the `mbed_lib.json` file unless `PSA` label is added to a target.